### ISSUE TITLE
Local Deployment of api-gateway does not generate time data in the "requestContext" of a lambda function event.

### ISSUE THREAD

**Issue by stevenj** [awslabs/aws-sam-cli#1902]
When a lambda function receives an event from the api-gateway running on AWS, it has:

```json
"requestContext": {
        "requestTime": "25/Mar/2020:04:53:40 +0000",
        "requestTimeEpoch": 1585112020858,
}
```

This information is completely missing from the "requestContext" of a locally deployed api-gateway.

**Comment by c2tarun** [comment]
Thanks for reporting this bug. [RequestContext](https://github.com/awslabs/aws-sam-cli/blob/ee91629e289eaa634809e6d691ce701426bab318/samcli/local/events/api_event.py#L65) is missing the fields required.

**Comment by mndeveci** [comment]
Patched with [v1.1.0](https://github.com/awslabs/aws-sam-cli/releases/tag/v1.1.0)
